### ISSUE TITLE
Learning rate scheduling for the diffopt

### ISSUE THREAD

**Issue by pratikgujjar** [facebookresearch/higher#73]
I am unclear on if this is possible at all. At the moment, I am running something like this:

```
opt = torch.optim.Adam(model.parameters(), lr=1e-3)
lrs = torch.optim.lr_scheduler.StepLR(opt, step_size=2, gamma=0.1)

for i in range(num_iterations):
    with higher.innerloop_ctx(model, opt, copy_initial_weights=True) as (fmodel, diffopt):
        logits = fmodel(X_train)
        loss = F.kl_div(F.log_softmax(logits, dim=1), F.softmax(y_train, dim=1), reduction='batchmean')
        diffopt.step(loss)

    lrs.step()
```

In the vanilla scenario higher is not really necessary. I have noticed discrepancy in training a model with diffopt versus the non higher case using a plain optimizer and the same scheduler. Is this the right way to go about scheduling learning rate for the differential optimizer?

**Comment by edchengg** [comment]
I guess the learning rate scheduler is quite important for BERT. Any ideas on how could I add it to the diffopt?

**Comment by hashemghanem** [comment]
A work-around in my case is to multiply the loss function in the inner-loop by the decaying factor. I don't know which decaying scheme you are using, but this work in most cases I guess.
